java/fundamentals/java-basics/functions.md

```diff
@@ -72,7 +72,7 @@ public class Example {
 ```
 ### Non-void Functions
 
-Instead of `void`, we could use primitive names if we want our function to return the result of a numbers multiplication or strings concatenation:
+Instead of `void`, we could use a concrete type to denote that our function will return a specific result. We could use an `int` to denote returning the result of integer multiplication or a `String` to denote returning the result of string concatenation. In general, we’re allowed to return any type available in Java (primitive or non-primitive).
```

Comment:
I'd add "for example" because this way it might sound like we can only use an `int` or a `String`

```suggestion
Instead of `void`, we could use a concrete type to denote that our function will return a specific result. For example, we could use an `int` to denote returning the result of integer multiplication or a `String` to denote returning the result of string concatenation. In general, we’re allowed to return any type available in Java (primitive or non-primitive).
```